doc/changelog.md

```diff
@@ -255,18 +396,29 @@
   shot in a shots-based execution mode.
```

Comment:
I think this wording is outdated, now that we do have the one-shot mode it is possible to produce an entire mcm array with it. 

Also might be worth considering as a new feature?

Reply:
https://github.com/PennyLaneAI/catalyst/pull/853/commits/3dc88f9fe0a77fef7bd1f20cb880f4fa20766261

Reply:
Nice addition 👍